shared/subprocess/proc.go

```diff
@@ -155,11 +156,11 @@ func (p *Process) start(fds []*os.File) error {
 	p.hasMonitor = true
 	go func() {
 		defer close(p.chExit)
-		/*if p.Stdout != nil {
```

Comment:
Looks like you're still working on this bit?

Reply:
The same as above.